Engine/TestPlanExecution.cs

```diff
@@ -589,6 +599,8 @@ private TestPlanRun DoExecute(IEnumerable<IResultListener> resultListeners, IEnu
             else
             {
                 execStage = new TestPlanRun(this, resultListeners.ToList(), initTime, initTimeStamp);
+                if (stepsOverride != null)
+                    execStage.Parameters.Add(new ResultParameter("StepListOverridden", "Yes"));
```

Comment:
Maybe this should be the list of guids in stepsOverride?

Reply:
I agree!